examples/javascript/default.mjs

```diff
@@ -0,0 +1,13 @@
+function square(a) {
```

Comment:
I think this file is misnamed: elsewhere you declare the suffix as `.js` (hence the check failure)

Reply:
So this one I named .mjs because the same linting rules as all other javascript files was being applied to the example file. However, the example file runs with d8 and uses `--allow-natives-syntax`.  This means there is a statement in there that the linter does not recognize. I thought it was okay to name it as a .mjs file since that just removes this file from the list of files to lint. And that is how all other example files are treated. However, I will see if I can instruct the linter to just ignore that statement.

Reply:
The reason I have named this file a `.mjs` is to get around the linter. The linter has an ignore pattern for files in the examples folder. However, it also throws a warning when it ignores that javascript file. Then since the max warnings is set to 0 I can not commit the file if its named as js. I get this error:
![image](https://user-images.githubusercontent.com/9499740/228748122-6848f9bb-de5d-4fcb-a136-2e94c259da6e.png)

Javascript files can be named with the extension `.mjs` : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules

I thought this is an acceptable workaround to this dilemma. However if you want me to fix this in another way, please suggest how I should go about it.

Reply:
I have updated the configuration in the language files to use the .mjs extension

Reply:
I don't see any failing checks now. Please tell me if I am missing something.

Reply:
Thanks : I understand what happened here. This workaround works for me!